lumigator/python/mzai/jobs/inference/requirements_cpu.txt

```diff
@@ -0,0 +1,11 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
Are we sure this is only used in tests? What gets installed for regular usage and what happens to people who do not have a GPU in that case?